doc/man1/openssl-ca.pod.in

```diff
@@ -336,6 +338,23 @@ If B<-multi-rdn> is not used then the UID value is C<123456+CN=John Doe>.
 
 This option generates a CRL based on information in the index file.
 
+=item B<-crl_lastupdate> I<time>
+
+This option allows the CRL's lastUpdate field to be explicitly set;
+if it is not present, the current time will be used. The format of the
+time is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure) or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timezone Z must be present.
+
+=item B<-crl_nextupdate> I<time>
+
+This option allows the value of the CRL's nextUpdate field to be
+explicitly set; if it is present, any values given for B<-crldays>,
+B<-crlhours> and B<-crlsec> will be ignored. The format of the time is
+YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure) or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timezone Z must be present.
+
```

Comment:
Maybe it's worth replacing the duplicated description of the format by the reference to the previous option?

Reply:
I followed the same style as `startdate` and `enddate` for consistency, but I agree that it looks too repetitive - I've simplified the documentation for the two new options.